src/lazy/binary/raw/value.rs

```diff
@@ -78,20 +116,160 @@ impl<'top> LazyRawValue<'top, BinaryEncoding_1_0> for LazyRawBinaryValue_1_0<'to
     fn read(&self) -> IonResult<RawValueRef<'top, BinaryEncoding_1_0>> {
         self.read()
     }
+}
 
-    fn range(&self) -> Range<usize> {
-        self.encoded_value.annotated_value_range()
+#[derive(Copy, Clone)]
+pub struct EncodedBinaryAnnotations_1_0<'a, 'top> {
+    value: &'a LazyRawBinaryValue_1_0<'top>,
+}
```

Comment:
🗺️ This type offers methods for accessing the various encoding components of a binary 1.0 annotations sequence.